strapi/design-system
# Dark mode/e2e

## What

- Added query URL to show dark mode
- Added dark mode e2e tests for every components
- Updated / improved stories for accessibility issues

## Test

- To check if story really opens in dark mode
- Add `headless: false` in `Playwright.config.js`
- Add .only for the dark mode test wanted
- Launch `yarn test:e2e` + complete path

## Issues

Components not accessible - tests commented for now:
- Button
- LinkButton
- DatePicker
- HeaderLayout (caused by Button)
- Layout (caused by Button)
